internal/tfr/getter.go

```diff
@@ -242,13 +242,18 @@ func getTerraformGetHeader(ctx context.Context, url url.URL) (string, error) {
 
 // getDownloadURLFromHeader checks if the content of the X-Terraform-GET header contains the base url
 // and prepends it if not
-func getDownloadURLFromHeader(ctx context.Context, url url.URL, terraformGet string) (string, error) {
+func getDownloadURLFromHeader(ctx context.Context, moduleURL url.URL, terraformGet string) (string, error) {
```

Comment:
I was wondering if `ctx` is required

Reply:
Ah good point. Removed in 776f8de

Reply:
Also noticed that one of the tests could be consolidated into the new table driven test, so did that in the same commit.